fv_be/app/models/characters.py

```diff
@@ -30,9 +31,13 @@ class Meta:
     notes = models.TextField(blank=True)
 
     # from fvcharacter:related_words
-    # TODO: add this as a foreign key to the dictionary entry model once it's created
-    # related_dictionary_entries = models.ForeignKey(
-    #     DictionaryEntry, on_delete=models.SET_NULL, blank=True, null=True, related_name="characters")
+    related_dictionary_entries = models.ForeignKey(
+        DictionaryEntry,
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        related_name="dictionary_entries",
```

Comment:
I believe the related name here would be something on the lines of "characters" or such. As it is what we would access from a dictionary entry with syntax such as `DictionaryEntry.characters.all()` to get the characters linked to a DictionaryEntry.

Reply:
Nice catch